netket/sampler/metropolis.py

```diff
@@ -525,3 +525,18 @@ def MetropolisHamiltonian(hilbert, hamiltonian, *args, **kwargs) -> MetropolisSa
 
     rule = HamiltonianRule(hamiltonian)
     return MetropolisSampler(hilbert, rule, *args, **kwargs)
+
+def MetropolisContinuous(hilbert, s, *args, **kwargs) -> MetropolisSampler:
+    r"""This sampler acts on all particle positions simultaneously and proposes a new state according to a Gaussian distribution with width s.
+    Args:
+       hilber: The continuous Hilbert space
+       s: The width of the Gaussian proposal distribution
+       n_chains: The number of Markov Chain to be run in parallel on a single process.
+       sweep_size: The number of exchanges that compose a single sweep.
+                   If None, sweep_size is equal to the number of degrees of freedom (n_visible).
+    """
+
+    from .rules import GaussianRule
+
+    rule = GaussianRule(s)
+    return MetropolisSampler(hilbert, rule, *args, **kwargs)
```

Comment:
For now I'd avoid having this and try to be explicit about the sampler/rule we are using.
This name is really too generic for what this is doing...
And in general we give the sampler the name of the rule (so in this case it would rather be `MetropolisGaussian`, which also ain't a great name)